parachain/primitives/ethereum/Cargo.toml

```diff
@@ -5,13 +5,16 @@ authors = ["Snowfork <contact@snowfork.com>"]
 edition = "2018"
 
 [dependencies]
-hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
-hex-literal = { version = "0.3.1", default-features = false }
-serde = { version = "1.0", optional = true }
 codec = { package = "parity-scale-codec", version = "1.3.4", default-features = false }
-rlp = { version = "0.4", default-features = false }
+ethbloom = { version = "0.8", default-features = false }
 ethereum-types = { version = "0.9.2", default-features = false }
+hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
+hex-literal = { version = "0.3.1", default-features = false }
 libsecp256k1 = { version = "0.3", default-features = false }
+parity-bytes = { version = "0.1", default-features = false }
+rlp = { version = "0.4", default-features = false }
+serde = { version = "1.0", optional = true }
+serde-big-array = { version = "0.2", optional = true }
```

Comment:
I sorted the dependencies but let me know if you folks would prefer I revert this to keep blame clean

Reply:
LGTM

Reply:
Makes sense. I'll refrain from sorting more stuff